manifests/params.pp

```diff
@@ -165,7 +165,7 @@
   $initial_location = undef
 
   $ipa_authentication = false
-  $http_keytab = '/etc/httpd/conf/http.keytab'
+  $http_keytab = undef
```

Comment:
Any reason not to use apache_conf_dir here directly?

Reply:
In params we haven't included apache yet, nor should we.